versions.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~> 3.17"
+      version = "~> 3.27"
```

Comment:
We actually don't want to bump the version as it's considered a breaking change. We are essentially limiting what version of 3 customers can be on. We should support as much of a range as possible. Are we using a new feature that requires use to bump this?

Reply:
> Are we using a new feature that requires use to bump this?

Actually, there's no need to bump it, I only tried to keep it up to date, but makes sense to keep it as is.